app/lib/theme/widgets/SelectRfpWidget.tsx

```diff
@@ -0,0 +1,81 @@
+import { WidgetProps } from "@rjsf/core";
+import { graphql, useFragment } from "react-relay";
+import SelectParentWidget from "./SelectParentWidget";
+
+interface EntitySchema {
+  list: [];
+  displayField: string;
+  placeholder: string;
+  label: string;
+}
+
+const SelectWidget: React.FunctionComponent<WidgetProps> = (props) => {
+  const { query } = useFragment(
+    graphql`
+      fragment SelectRfpWidget_query on Query {
+        query {
+          allFundingStreams {
+            edges {
+              node {
+                rowId
+                name
+                description
+              }
+            }
+          }
+          allFundingStreamRfps {
+            edges {
+              node {
+                rowId
+                year
+                fundingStreamId
+              }
+            }
+          }
+        }
+      }
+    `,
+    props.formContext.query
+  );
+
+  const parent = {
+    list: query.allFundingStreams.edges.map((edge) => {
+      const { rowId, name, description } = edge.node;
+      return {
+        rowId: rowId,
+        name: name,
+        description: description,
+      };
+    }),
+    displayField: "name",
+    placeholder: "Select a Funding Stream",
+    label: "Funding Stream",
+  } as EntitySchema;
+
+  const child = {
+    list: query.allFundingStreamRfps.edges.map((edge) => {
+      const { rowId, year, fundingStreamId } = edge.node;
+      return {
+        rowId: rowId,
+        year: year,
+        fundingStreamId: fundingStreamId,
+      };
+    }),
+    displayField: "year",
+    placeholder: "Select a Funding Stream RFP Year",
+    label: "Funding Stream RFP",
+  } as EntitySchema;
```

Comment:
You'll want some `useMemo` hooks to avoid creating a new list of options at every render